components/admin_console/billing/billing_subscriptions/billing_subscriptions.tsx

```diff
@@ -91,10 +110,18 @@ export const contactSalesCard = (contactSalesLink: any, isFreeTrial: boolean, tr
                     className='PrivateCloudCard__contactSales'
                     onClick={() => trackEvent('cloud_admin', 'click_contact_sales')}
                 >
-                    <FormattedMessage
-                        id='admin.billing.subscription.privateCloudCard.contactSales'
-                        defaultMessage='Contact Sales'
-                    />
+                    {subscriptionPlan !== 'CLOUD_STARTER' &&
+                        <FormattedMessage
+                            id='admin.billing.subscription.privateCloudCard.contactSales'
+                            defaultMessage='Contact Sales'
+                        />
+                    }
+                    {subscriptionPlan === 'CLOUD_STARTER' &&
+                        <FormattedMessage
+                            id='admin.billing.subscription.privateCloudCard.upgradeNow'
+                            defaultMessage='Upgrade Now'
+                        />
+                    }
                 </a>
             </div>
             <div className='PrivateCloudCard__image'>
```

Comment:
Super minor detail: There is a new image for the trial version that doesn't have the lock. See https://www.figma.com/file/B3YyrLCLAjNfdPUNq0shQq/Cloud-Professional-Trial-Flow?node-id=103%3A227350